src/index.js

```diff
@@ -1,14 +1,110 @@
 import _ from 'lodash';
 import './style.css';
+import refreshImage from './refresh.png';
+import arrow from './arrow-key.svg';
+import dots from './dots.svg';
+import trash from './trash.svg';
 
- function component() {
-   const element = document.createElement('div');
+// selecting the container
+const container = document.querySelector('.container');
 
-  // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-   element.classList.add('hello');
+// selecting the header section
+const headerSection = document.querySelector('.header');
 
-   return element;
- }
+// making the refresh button image
+const refresh = document.createElement('img');
+refresh.classList.add('refreshButton');
+refresh.src = refreshImage;
 
- document.body.appendChild(component());
\ No newline at end of file
+headerSection.append(refresh);
+
+// selecting the input container
+const inputSection = document.querySelector('.inputSection');
+
+// selecting the input field
+const inputField = document.getElementById('inputField');
+
+const arrowHolder = document.createElement('img');
+arrowHolder.classList.add('arrow');
+arrowHolder.src = arrow;
+
+inputSection.append(inputField, arrowHolder);
+
+// selecting the button
+const button = document.querySelector('.clearAll');
+
+// implementing list functionality
+
+const taskHolder = [];
```

Comment:
- [ ] As requested by the previous reviewer, it is expected that you render a created list of tasks dynamically per the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md). I highly suggest you populate your `const taskHolder = [];` which is an empty array with data.
This is an example of how to populate the data:

```
const taskHolder = [
  { description: 'task1', completed: false, index: 1 },
    { description: 'task2', completed: true, index: 2 },
    { description: 'task3', completed: false, index: 3 },
];

```

This is a screenshot of how the populated data should render dynamically. 😄 

![Screenshot (296)](https://github.com/gisachris/To-Do-List/assets/78104544/ea0a00fa-c751-4e6d-9685-5f917e4d7406)

This is a screenshot of your project. 😄 

![Screenshot (298)](https://github.com/gisachris/To-Do-List/assets/78104544/3fc7d407-fd78-4a7d-aa26-74a048558d5f)

Reply:
ok thankyou the previous reviewer didn't clearly specify what needed to be done 👍

Reply:
> ok thankyou the previous reviewer didn't clearly specify what needed to be done 👍

You welcome